webpack.config.ts

```diff
@@ -56,10 +56,10 @@ function parseFeatureDetails(readmeContent: string, id: FeatureID): FeatureMeta
 }
 
 function getFeatures(): FeatureID[] {
-	return Array.from(
-		readFileSync(path.join(__dirname, 'source/refined-github.ts'), 'utf-8').matchAll(/^import '\.\/features\/([^.]+)';/gm),
-		match => match[1] as FeatureID
-	).sort().filter(id => !id.startsWith('rgh-'));
+	return [...readFileSync(path.join(__dirname, 'source/refined-github.ts'), 'utf-8').matchAll(/^import '\.\/features\/([^.]+)';/gm)]
```

Comment:
Let’s move the readFileSync to it’s own constant. Match all can stay on this line probably

Reply:
I have to go now. I can work on it tomorrow or it you want you can commit.

Reply:
No rush